package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "conseiljs",
-  "version": "0.1.25",
+  "version": "0.1.29-z",
   "description": "Client-side library for dApp development.",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha --timeout 10000 -r ts-node/register test/**/*.spec.ts",
+    "test": "./node_modules/mocha/bin/mocha --timeout 10000 -r ts-node/register test/*.spec.ts",
```

Comment:
Since we moved the integration tests to a different directory, it's best to revert this change.